src/components/PostsContainer/PostsContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Post } from '../Post/Post';
+import useMessages from '../../hooks/useMessages';
+import useAuthToken from '../../hooks/useAuthToken';
+//todo import { NavigationButtons } from '../NavigationButtons/NavigationButtons';
+
+export const PostsContainer: React.FC = ({}) => {
+  const token = useAuthToken();
+  const messages = useMessages(token);
```

Comment:
you can simplify this part of the code with using `useAuthToken();`  inside the `useMessages` hook!